meta-balena-common/recipes-core/systemd/timeinit/fake-hwclock-update.service

```diff
@@ -13,14 +13,9 @@
 # limitations under the License.
 
 [Unit]
-Description=Initialize system clock from build timestamp
+Description=Update the saved fake-hwclock file.
 DefaultDependencies=no
-Before=chronyd.service
 
 [Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStart=/bin/sh /usr/bin/timeinit-timestamp.sh
-
-[Install]
-WantedBy=time-sync.target
+Type=simple
```

Comment:
`simple` implies a supervised daemon, from what I understand the script exits after it does its thing so shouldn't this be `oneshot`?

Reply:
I used `simple` as it is non-blocking and we don't need to block in this case. As this is not running as part of the boot process it may not make much difference which one is used.